docs/python/src/model/model.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import psutil
+
+from PySide2.QtGui import QGuiApplication
+from PySide2.QtQml import QQmlApplicationEngine, qmlRegisterType
+from PySide2.QtCore import QUrl, QTimer, QAbstractListModel
+
+from PySide2.QtCore import QObject, Signal, Slot, Property, Qt
+
+class CpuLoadModel(QAbstractListModel):
+    def __init__(self):
+        QAbstractListModel.__init__(self)
+        
+        self._cpu_count = psutil.cpu_count()
+        self._cpu_load = [0] * self._cpu_count
+        
+        self._updateTimer = QTimer(self)
+        self._updateTimer.setInterval(1000)
+        self._updateTimer.timeout.connect(self._update)
+        self._updateTimer.start()
+        
+        # The first call returns invalid data
+        psutil.cpu_percent(percpu=True)
+            
+    def _update(self):
+        self._cpu_load = psutil.cpu_percent(percpu=True)
+        self.dataChanged.emit(self.index(0,0), self.index(self._cpu_count-1, 0))
+        
+    def rowCount(self, parent):
+        return self._cpu_count
+    
+    def data(self, index, role):
+        if role == Qt.DisplayRole and \
```

Comment:
Try to avoid always extending lines with `\`, instead please use the round parenthesis, so you can rewrite this if conditions like:
```
if (role == Qt.DisplayRole and
     index.row() >= 0 and
     index.row() < len(self._cpu_load) and
     index.column() == 0):
```

Reply:
Fixed!